### PR TITLE
First aid kit tests

### PR DESCRIPTION
**Pull request by nelealbers**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/318: end to end tests for first aid kit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
     "cucumber": "^7.0.0-rc.0",
     "dotenv": "^16.0.1",
     "isomorphic-fetch": "^3.0.0",
-    "pg": "^8.8.0"
+    "pg": "^8.9.0"
```
Comment (nelealbers) [inline package.json:7]: @wbaccinelli I am not sure why this is different, I do not recall changing anything here.

File: features/step_definitions/agenda_steps.js [hunk 2]
```diff
@@ -50,6 +50,20 @@ function verifyRasaResponse(expected_response, callback){
...
+  assert(context.therapist_response.hasOwnProperty("content"));
+  assert(context.therapist_response['content'].hasOwnProperty("TEXT"));
+  var matching_text = 'default_text';
+  for (var i = 0; i < expected_response.length; i++){
+	  if (context.therapist_response['content']['TEXT'].includes(expected_response[i])){
+		  matching_text = expected_response[i];
+	  }
+  }
+  assert(context.therapist_response['content']['TEXT'].includes(matching_text));
+  callback();
```
Comment (wbaccinelli) [inline features/step_definitions/agenda_steps.js:64]:
```suggestion
  for (var i = 0; i < expected_response.length; i++){
	  if (context.therapist_response['content']['TEXT'].includes(expected_response[i])){
		  callback();
	  }
  }
  callback('Error: ', context.therapist_response['content']['TEXT']);
```

File: package.json (hunk 1 above)
Comment (wbaccinelli) [reply package.json:7]: Let's not push this then

**Review by wbaccinelli — APPROVED** [review]

**Comment by wbaccinelli** [comment]
Good job @nelealbers !

File: package.json (hunk 1 above)
Comment (nelealbers) [inline package.json:7]:
```suggestion
    "pg": "^8.8.0"
```

**Pull request merged**